docs/connect.txt

```diff
@@ -179,6 +179,10 @@ The CrateDB JDBC driver supports following properties:
      authenticate as the CrateDB superuser, which is ``crate``. The superuser
      does not have a password, so you can omit the ``password`` property.
 
+     If you are authenticating as a regular user, that use will need to have
```

Comment:
not sure but: section above talks about a "custom database user" and here it is called a "regular user". maybe we should call it the same to be consistent and not confusing?